src/azure-cli/azure/cli/command_modules/iot/tests/hybrid_2019_03_01/test_iot_commands.py

```diff
@@ -140,9 +140,7 @@ def test_iot_hub(self, resource_group, resource_group_location, storage_account)
             hub, policy_name, policy['primaryKey'])
 
         # Test policy_name connection-string 'az iot hub show-connection-string'
-        self.cmd('iot hub show-connection-string -n {0} --policy-name {1}'.format(hub, policy_name), checks=[
-            self.check_pattern('connectionString', policy_name_conn_str_pattern)
-        ])
+        self.cmd('iot hub show-connection-string -n {0} --policy-name {1}'.format(hub, policy_name))
```

Comment:
Why you change the test_iot_commands.py file?

Reply:
If you want to record the test_iot_hub test for profile 2019-03-01, please checkout to that profile first `az cloud set --name AzureCloud --profile 2019-03-01-hybrid` and run test in live again.

Reply:
Why you want to re-record this test? Is this test related with eventhub module?